site/src/js/index.js

```diff
@@ -4,13 +4,39 @@
 let contentEl = document.querySelector('#content'),
     filterInput = document.querySelector('#filter'),
     sortingInput = document.querySelector('#sorting'),
+    claps = {},
     firstLoad = true,
     selector = new Selectr('#tags', {
         multiple: true,
         placeholder: 'Choose tags...',
         data: window.swagTags.map(tag => ({value: tag, text: tag}))
     });
 
+const API = 'https://api.applause-button.com';
+
+const requestClaps = (urls) => {
+    return new Promise((resolve) => {
+        fetch(`${API}/get-multiple`, {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json; charset=utf-8',
+            },
+            referrer: 'devswag.io',
```

Comment:
The `Referer` HTTP header needs an URL (with it's http* prefix): https://tools.ietf.org/html/rfc1945#section-10.13

Not sure it's really needed as it looks to be only used as a key.